Python/Heart Disease Prediction/README.md

```diff
@@ -0,0 +1,25 @@
+# Heart disease prediction
+This python script uses a machine learning model to predict whether a person has heart disease or not.The Model is trained with [ucl heart disease dataset](https://archive.ics.uci.edu/ml/datasets/heart+disease)
+
+# How to run program
+**1.**  Fork [this](https://github.com/HarshCasper/Rotten-Scripts) repository.
+
+**2.**  Clone your forked copy of the project.
+```
+git clone https://github.com/<your-user-name>/Rotten-Scripts.git
+```
+Navigate to the project directory.
+```bash
+cd \Rotten-Scripts\Python\Heart Disease Prediction
+```
+Install the requirements 
+```bash
+pip install -r requirements.txt
+```
+## To run
+```bash
+python script.py
+```
+## Author
+ 
+[Adithya Kurien Peter](https://github.com/aaadddiii)
```

Comment:
Dear @aaadddiii your script shares similarity with [this](https://github.com/mrdbourke/zero-to-mastery-ml/tree/master/section-3-structured-data-projects).

I am all in for inspired code. I have done the same many times, but am against using someone else's code and claiming it as it's your own.

You can do either of these 3 things

1. in the README mention the exact name of the Author
2. Make some changes in the existing code to make it better
3. Add some personal touches to the script

Reply:
Sure

Reply:
The similarity of the code is that i learned machine learning from his course this is a project in his course . i'l mention his name in the README .

Reply:
That seems good enough to me.

Kindly make the changes in the README, make sure that you properly mention the source as well as your contribution.